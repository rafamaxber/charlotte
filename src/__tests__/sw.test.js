import React from 'react';
import Sw from '../registerServiceWorker';

describe('registerServiceWorker', () => {
  it('renders correctly', () => {
    expect(Sw).toMatchSnapshot();
  });
});
