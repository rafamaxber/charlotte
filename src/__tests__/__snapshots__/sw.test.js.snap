// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`registerServiceWorker renders correctly 1`] = `[Function]`;
